devtools/cli/src/main/java/io/quarkus/cli/CreateCli.java

```diff
@@ -67,14 +68,9 @@ public Integer call() throws Exception {
                     output, propertiesOptions.properties);
 
             boolean success = true;
-
-            // TODO: default extension (picocli)
-
             if (runMode.isDryRun()) {
                 createProject.dryRun(buildTool, invocation, output);
-            } else if (buildTool == null) { // buildless / JBang
-                success = new CreateJBangProjectCommandHandler().execute(invocation).isSuccess();
-            } else { // maven or gradle
+            } else {
```

Comment:
I understand that `CreateProjectCommandHandler` is able to handle the JBang case?

Reply:
It isn't so much that, as the runner used after that makes the appropriate decision such that the special case isn't required. (buildTool will never be null, and there is one for JBang now)

Reply:
Updated (tests for maven, grade, and jbang. Uncovered an issue with the jbang picocli template)